p2p_wallet/Coordinators/SolendDepositCoordinator.swift

```diff
@@ -26,12 +26,14 @@ final class SolendDepositCoordinator: Coordinator<Void> {
         super.init()
     }
 
-    override func start() -> AnyPublisher<Void, Never> {
-        let resultSubject = PassthroughSubject<Void, Never>()
+    override func start() -> AnyPublisher<Bool, Never> {
+        let resultSubject = PassthroughSubject<Bool, Never>()
 
         let viewModel = try! DepositSolendViewModel(strategy: initialStrategy, initialAsset: initialAsset)
         viewModel.finish.sink { [unowned self] in
             controller.popViewController(animated: true)
+            resultSubject.send(true)
+            resultSubject.send(completion: .finished)
```

Comment:
better to not send completion, but

return resultSubject.prefix(1).eraseToAnyPublisher()